### PR TITLE
core: add ability to update account tags

### PR DESCRIPTION
**Pull request by jeffomatic**

### CODE REVIEW

**Comment by jeffomatic** [comment]
PTAL, ran it using batch and singleton variants of this script: https://gist.github.com/jeffomatic/49e8f6a68e500009122d700226936d40

**Comment by kr** [comment]
:eyes:

File: core/account/accounts.go [hunk 1]
```diff
@@ -128,6 +131,67 @@ func (m *Manager) Create(ctx context.Context, xpubs []chainkd.XPub, quorum int,
...
+		}
+		alias = &a
+	} else {
+		signer, err = m.FindByAlias(ctx, *alias)
+		if err != nil {
+			return errors.Wrap(err, "get account by alias")
+		}
+	}
+
+	if err != nil {
```
Comment (kr) [inline core/account/accounts.go]: This seems to be superfluous?

Comment (jeffomatic) [inline core/account/accounts.go]: Yes, it's a legacy of a previous iteration. Nice catch!

**Pull request merged**